docs/TUTORIAL.md

```diff
@@ -25,17 +25,17 @@
 ### Overview ###
 A software update system must follow two steps to integrate The Update
 Framework (TUF).  First, it must add the framework to the client side of the
-update system.  The [tuf.client.updater](client/README.md) module assists in
+update system.  The [tuf.client.updater](../tuf/client/README.md) module assists in
 integrating TUF on the client side.  Second, the software repository on the
 server side must be modified to include a minimum of four top-level metadata
 (root.json, targets.json, snapshot.json, and timestamp.json).  No additional
 software is required to convert a software repository to a TUF one.  The
 low-level repository tool that generates the required TUF metadata for a
 software repository is the focus of this tutorial.  There is also separate
 document that [demonstrates how TUF protects against malicious
-updates](ATTACKS.md).
+updates](SECURITY.md).
```

Comment:
We actually want a link to `ATTACKS.md,` which demonstrates how the code protects against the attacks outlined in `SECURITY.md.`

Here is the full link: https://github.com/theupdateframework/tuf/blob/develop/tuf/ATTACKS.md

Reply:
Good catch. I substituted `SECURITY.md` in because I didn't see `ATTACKS.md` somehow.

Reply:
Fixed in commit c5dd553